test/NuGet.Clients.Tests/NuGet.CommandLine.Test/CommandRunner.cs

```diff
@@ -29,6 +26,7 @@ public class CommandRunner
                 RedirectStandardError = true,
                 RedirectStandardInput = inputAction != null
             };
+            psi.EnvironmentVariables["NuGetTestModeEnabled"] = "True";
```

Comment:
Also, there's this comment, not sure if taken care of: https://github.com/NuGet/NuGet.Client/pull/135#issuecomment-159367341